### PR TITLE
Debug logging

### PR DESCRIPTION
**Pull request by ddsharpe**
Increase the documentation for debug logging.

### CODE REVIEW

**Review by jshum2479 — APPROVED** [review]

File: site/logging.md [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
+#handlers=java.util.logging.ConsoleHandler
+handlers=java.util.logging.FileHandler, java.util.logging.ConsoleHandler
+```
+Then, change the Level to `FINE` or `FINER`.
+```properties
+#com.oracle.weblogic.imagetool.level=INFO
+com.oracle.weblogic.imagetool.level=FINER
+```
+
+Logging levels from highest to lowest:
```
Comment (rosemarymarano) [inline site/logging.md:19]: Logging levels -> Logging severity levels

File: site/logging.md [hunk 2]
```diff
@@ -1,8 +1,31 @@
...
+
+| Level | Description |
+| --- | --- |
+| `SEVERE` | Only error messages will be written to the log file. |
+| `WARNING` | Warning messages and above will be written to the log file. |
```
Comment (rosemarymarano) [inline site/logging.md:24]: and above -> and higher; also make this change in the next three descriptions.

File: site/logging.md [hunk 3]
```diff
@@ -1,8 +1,31 @@
...
+| `INFO` | (Default) Informational messages and above will be written to the log file. |
+| `FINE` | First level of debug messages and above are written to the log file. |
```
Comment (rosemarymarano) [inline site/logging.md:26]: First level of debug messages -> First level debug messages

File: site/logging.md [hunk 4]
```diff
@@ -1,8 +1,31 @@
...
+```properties
+#com.oracle.weblogic.imagetool.level=INFO
+com.oracle.weblogic.imagetool.level=FINER
+```
+
+Logging levels from highest to lowest:
+
+| Level | Description |
+| --- | --- |
+| `SEVERE` | Only error messages will be written to the log file. |
```
Comment (rosemarymarano) [inline site/logging.md:23]: will be -> are; also make this change in the next two descriptions

File: site/logging.md [hunk 5]
```diff
@@ -1,8 +1,31 @@
...
+| `WARNING` | Warning messages and above will be written to the log file. |
+| `INFO` | (Default) Informational messages and above will be written to the log file. |
+| `FINE` | First level of debug messages and above are written to the log file. |
+| `FINER` | Detailed debug messages and above are written to the log file. |
+| `FINEST` | All HTTP responses in addition to `FINER` level debug messages are written to the log file. |
```
Comment (rosemarymarano) [inline site/logging.md:28]: All HTTP responses in addition to `FINER` level debug messages are written to the log file. -> All HTTP responses, in addition to `FINER` level debug messages, are written to the log file. OR In addition to `FINER` level debug messages, all HTTP responses are written to the log file.

**Review by rosemarymarano — COMMENTED** [review]
Mostly suggested edits.

File: site/logging.md [hunk 6]
```diff
@@ -1,8 +1,31 @@
...
 
-The Image Tool uses the standard `logging.properties` file under the tool's ```bin``` directory.
+### To enable debug logging
+In `logging.properties`, comment the existing `handlers` property and uncomment the `handlers` line below the first that 
+contains the FileHandler.
+```properties
+#handlers=java.util.logging.ConsoleHandler
+handlers=java.util.logging.FileHandler, java.util.logging.ConsoleHandler
+```
+Then, change the Level to `FINE` or `FINER`.
```
[the remark this answers is not in the document]
Comment (ddsharpe) [reply site/logging.md:13]: Level is a proper name of the object.  java.util.logging.Level.  Level is an enum class.

**Review by rosemarymarano — APPROVED** [review]

**Review by mriccell — APPROVED** [review]

**Pull request merged**